chroot-script

```diff
@@ -269,14 +269,21 @@ packages() {
   }
 
   if [ "$PACKAGES" = 'yes' ] ; then
-     if ! [ -r /etc/debootstrap/packages ] ; then
-       echo "Error: /etc/debootstrap/packages (inside chroot) not found, exiting." >&2
-       exit 1
-     else
-       $APTUPDATE
-       # shellcheck disable=SC2086,SC2046
-       DEBIAN_FRONTEND=$DEBIAN_FRONTEND $APTINSTALL $(grep -v '^#' /etc/debootstrap/packages) $GRMLPACKAGES
-     fi
+      PACKAGES_FILE="/etc/debootstrap/packages"
+
+      if [ "$ARCH" = 'arm64' ]; then
+        PACKAGES_FILE="/etc/debootstrap/packages-arm64"
+      fi
+
+      if ! [ -r $PACKAGES_FILE ] ; then
```

Comment:
nit: "${PACKAGES_FILE}" (to have "${...}" style variable and double-quotes to not break if the file name should include white spaces)